src/docs/product/integrations/notification-incidents/signl4/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: SIGNL4
+sidebar_order: 1
+redirect_from:
```

Comment:
I don't think you need this redirect here - seems to be causing issues

Reply:
Looks good to me after https://github.com/getsentry/sentry-docs/pull/6561/commits/7b1f26cfdc184d187abc2e09af756c12eb34ccee